recipe/install_runtime.bat

```diff
@@ -45,10 +45,11 @@ if %ERRORLEVEL% GTR 8 exit 1
 robocopy "%REDIST_ROOT%\Microsoft.VC%VC_VER%.OpenMP" "%PREFIX%" *.dll /E
 if %ERRORLEVEL% GTR 8 exit 1
 
-REM ========== REQUIRES Win 10 SDK be installed, or files otherwise copied to location below!
-robocopy "C:\Program Files (x86)\Windows Kits\10\Redist\ucrt\DLLs\%VC_PATH%"  "%LIBRARY_BIN%" *.dll /E
+REM ========== REQUIRES Win 10/11 SDK be installed, or files otherwise copied to location below!
+C:\Program Files (x86)\Windows Kits\10\Redist\%sdk_version%\ucrt\DLLs\%VC_PATH%
```

Comment:
Is this line a part of the comment?

Reply:
Thanks, I removed it, that was added by mistake.